examples/xds-management-server/config.yaml

```diff
@@ -0,0 +1,15 @@
+clusters:
```

Comment:
Add a comment at the top, stating that this is the source for a file provider?

I was staring at this for a bit trying to work out what it was for 😁